README.md

```diff
@@ -128,21 +128,37 @@ dependencies {
 }
 ```
 
-**Tomcat 8.x:**
+**Tomcat 8.0.x:**
 
 ```groovy
 repositories {
     mavenCentral()
 }
 
 dependencies {
-    def tomcatVersion = '8.5.12'
+    def tomcatVersion = '8.0.42'
     tomcat "org.apache.tomcat.embed:tomcat-embed-core:${tomcatVersion}",
            "org.apache.tomcat.embed:tomcat-embed-logging-juli:${tomcatVersion}",
            "org.apache.tomcat.embed:tomcat-embed-jasper:${tomcatVersion}"
 }
 ```
 
+**Tomcat 8.5.x:**
+
+```groovy
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    def tomcatVersion = '8.5.12'
+    tomcat "org.apache.tomcat.embed:tomcat-embed-core:${tomcatVersion}",
+           "org.apache.tomcat.embed:tomcat-embed-logging-juli:8.5.2",
```

Comment:
Hi @bmuschko, tomcat-embed-logging-juli - the latest version on Maven Central is 8.5.2. I don't know if it is required - I have just updated the existing documentation.